docusaurus/docs/cloud/advanced/upload.md

```diff
@@ -0,0 +1,204 @@
+
+Examples of configuring the `contentSecurityPolicy` depending on upload provider:
+
+<Tabs groupId="upload-examples" >
+<TabItem value="cloudinary" label="Cloudinary">
+
+```js
+module.exports = [
+  // ...
+  {
+    name: 'strapi::security',
+    config: {
+      contentSecurityPolicy: {
+        useDefaults: true,
+        directives: {
+          'connect-src': ["'self'", 'https:'],
+          'img-src': ["'self'", 'data:', 'blob:', 'market-assets.strapi.io', 'res.cloudinary.com'],
+          'media-src': [
+            "'self'",
+            'data:',
+            'blob:',
+            'market-assets.strapi.io',
+            'res.cloudinary.com',
+          ],
+          upgradeInsecureRequests: null,
+        },
+      },
+    },
+  },
+  // ...
+];
+```
+</TabItem>
+<TabItem value="amazon-s3" label="Amazon S3">
+
+```js
+module.exports = [
+  // ...
+  {
+    name: 'strapi::security',
+    config: {
+      contentSecurityPolicy: {
+        useDefaults: true,
+        directives: {
+          'connect-src': ["'self'", 'https:'],
+          'img-src': [
+            "'self'",
+            'data:',
+            'blob:',
+            'market-assets.strapi.io',
+            'yourBucketName.s3.yourRegion.amazonaws.com',
+          ],
+          'media-src': [
+            "'self'",
+            'data:',
+            'blob:',
+            'market-assets.strapi.io',
+            'yourBucketName.s3.yourRegion.amazonaws.com',
+          ],
+          upgradeInsecureRequests: null,
+        },
+      },
+    },
+  },
+  // ...
+];
+```
+</TabItem>
+</Tabs>
+
+
+
+
+
+:::caution
+The upload provider must be installed as a package dependency in your Strapi project. For example, to use the Cloudinary upload provider, install the `@strapi/provider-upload-cloudinary` package.
+
+For specific configuration information, see the [Marketplace](https://market.strapi.io/) entry for that provider.
+:::
+
+
+
+## Strapi Cloud Configuration
+Before pushing changes to GitHub, add environment variables to the Strapi Cloud project:
+
+1.  Log into Strapi Cloud and click on the corresponding project on the Projects page.
+2.  Click on the **Settings** tab and choose **Variables** in the left menu.
+3.  Add the required environment variables specific to the upload provider.
+4.  Click **Save**.
+
+### Cloudinary Example:
+
+| Variable | Value |
+| -------- | ----- |
+| `CLOUDINARY_NAME` | your_cloudinary_name |
+| `CLOUDINARY_KEY` | your_cloudinary_api_key |
+| `CLOUDINARY_SECRET` | your_cloudinary_secret|
+
+## Deployment
+
+To deploy the project and utilize the third-party upload provider, push the changes from earlier. This will trigger a rebuild and new deployment of the Strapi Cloud project.
+
+Once the application finishes building, the project will use the new upload provider.
+
+:::strapi
+For information on creating a custom upload provider, see the [Providers](/dev-docs/providers#creating-providers) documentation.
+:::
```

Comment:
Same feedback as in the other doc, would be nice to add a title & rephrase a tiny bit.